src/types/cdn.ts

```diff
@@ -1,2 +1,6 @@
+// TODO: v11 Remove
+/** @deprecated Use DiscordImageSize */
 export type ImageSize = 16 | 32 | 64 | 128 | 256 | 512 | 1024 | 2048;
+// TODO: v11 Remove
+/** @deprecated Use DiscordImageFormat */
```

Comment:
The API types are for internal use, it is not recommended to use the types exclusively... so we can just delete the entire file?

Reply:
IMO we can keep this as it's not really causing any difficulty/issues and can easily be nuked once we ready for v11. Before we do it we just search TODO: v11 and we will find all the stuff we want done for it.

This prevents even the tiniest of breaking change and tbh i would love for a librarby to handle breaking changes with this much care. Even the smallest of breaks is a breaking change. Atm nothing on this is marking it private and therefore technically speaking it is open to the public api.